packages/react-native/Libraries/LogBox/UI/LogBoxInspectorStackFrame.js

```diff
@@ -70,6 +70,7 @@ function getFileName(file: ?string) {
 
 const styles = StyleSheet.create({
   frameContainer: {
+    direction: 'ltr',
```

Comment:
Why is this one also forced to be LTR?

Reply:
It is used for Call Stack style and is not related to AnsiHighlight style. I will revert it.